Nomster/AppDelegate.swift

```diff
@@ -15,9 +15,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var suggestions: NSMutableArray = []
 
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
-        let path = NSBundle.mainBundle().pathForResource("data", ofType: "json")
-        let data = NSData(contentsOfFile: path!)
-        var results = NSJSONSerialization.JSONObjectWithData(data!, options: NSJSONReadingOptions.MutableContainers, error: nil) as! NSArray
+        
+        //let loader: FixturesLoader = FixturesLoader()
+        let loader: RESTLoader = RESTLoader()
+
```

Comment:
Technically not. But I wanted to optical separate the loader initialization because it should be possible in future to switch between fixture- and rest-loader.